### PR TITLE
If electron-webrtc is required in the browser, return the browser RTC implementation

### PR DESCRIPTION
**Pull request by feross**

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var EventEmitter = require('events').EventEmitter
+var getBrowserRTC = require('get-browser-rtc')
+
+module.exports = function (opts) {
+  var wrtc = new EventEmitter()
+  // If electron-webrtc is required in the browser, return the browser RTC implementation.
+  return Object.assign(wrtc, getBrowserRTC(), {
```
Comment (mappum) [inline browser.js]: Maybe we should get the `assign` function from the `object-assign` package? Unfortunately some browsers don't support `Object.assign`.

Comment (feross) [inline browser.js]: If you insist, I'll add it. But I don't think it's a big deal. Browsers that support webrtc all support `Object.assign`.

The only browser this excludes is IE11 and below. v

**Comment by feross** [comment]
This makes it easier for me to test `simple-peer` with `electron-webrc` as the backing implementation. This is not strictly required, but it's the same thing that `node-webrtc` does if you require it.

File: browser.js (hunk 1 above)
Comment (mappum) [inline browser.js]: Ah, nevermind then. Last time I checked it wasn't supported in Safari so I have made a habit of using `object-assign`, but it seems that has changed.

**Comment by feross** [comment]
Sweet, thanks!

**Comment by feross** [comment]
@mappum npm release, pretty please?

**Comment by mappum** [comment]
@feross Published as `electron-webrtc@0.3.0`, and will also add you as a collaborator

**Comment by feross** [comment]
👏

**Pull request merged**